### ISSUE TITLE
[BUG]: In the checker report, JAWS reading the wrong thing

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#1682]
### Project

extension-checker

### Browser

Chrome

### Operating system

Windows

### Description

When I did a scan, and then filtered issues, JAWS would not correctly read the issue with focus. It would read some recommendation when a violation has focus. Re-scanning didn't appear to help, but collapsing and expanding the row did. Seems that JAWS is caching something and not recognizing the update. We need to understand why, and perhaps file a bug with JAWS.

May need to re-check this after the table re-work.

### Steps to reproduce

* While running JAWS, scan a page. Then, uncheck the "Needs Review" and "Recommendation" checkboxes.
* Tab to the treegrid with the violations
* Arrow down through the violations under one announces a recommendation

Note that if you item down through the content in item mode, it reads the correct thing. If you collapse and expand, it reads the correct thing.

**Comment by shunguoy** [comment]
triage: need to varify if it's a bug in JAWS or in our code (e.g., event).

**Comment by shunguoy** [comment]
Triage: can be duplicated with JAWS in Windows.

**Comment by shunguoy** [comment]
triage: need to confirm with NVDA to see if it's a JAWS bug.  If it's a JAWS bug, refresh the buffer to try.

**Comment by philljenkins** [comment]
Using the F5 command, it Refreshes the page and causes the need to re-scan the page

**Comment by philljenkins** [comment]
@tombrunet is there also a Carbon component that exhibits the same issue? link to storyboard?

I did not see a filter option example in the ARIA APG Advanced Data Grid https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/advanced-data-grid/

**Comment by shunguoy** [comment]
tested in Voice Over. looks like Voice Over doesn't have the issue.

**Comment by shunguoy** [comment]
Triage: open an issue with JAWS using our page as a reference/test page.  Phil will test with NVDA, and if it's ok to open JAWS issue. @philljenkins

**Comment by philljenkins** [comment]
NCDA installed and testing

**Comment by philljenkins** [comment]
Upgraded to JAWS 2024.
Testing with JAWS 2024 does NOT have the previously reported bug of reading the wrong issue (row) that has focus.  

JAWS read the correct row, the # of issues in that group. and whether it can be collapsed or expanded.  The counts are correct depending if it is a "group" or one or more of the issues listed in a group.  

I tested with all three views: by Element, Requirement, and Rule.

Procedure I followed:
Tab to issue list, then arrow Up/Down through issue list. (Using Tab will move outside of issue list)

**Comment by philljenkins** [comment]
So either JAWS 2024 fixed the tracking problem, or the fixes to the "data grid" table of issues fixed the problem.

**Comment by philljenkins** [comment]
A new issue was discovered when closing this issue
#1771
